src/index.js

```diff
@@ -0,0 +1,110 @@
+//GET THE AXIOS CALL WORKING
+//LOOK AT THE 7-WONDERS CODE TO TRY TO GET THE PROMISE AND THE ASYNC AND AWAIT FUNCTION WORKING.
+
+//CODE FOR BUTTON:
+
+const state = {
+  tempCount: 60, //was temp
+};
+var tempCountContainer = document.querySelector('#tempCount');
+var gardenContainer = document.querySelector('#garden');
+
+//changing the background color temp function
+const setTempColor = () => {
+  if (state.tempCount <= 49) {
+    tempCountContainer.style.backgroundColor = 'teal';
+    gardenContainer.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (state.tempCount >= 50 && state.tempCount <= 59) {
+    tempCountContainer.style.backgroundColor = 'green';
+    gardenContainer.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (state.tempCount >= 60 && state.tempCount <= 69) {
+    tempCountContainer.style.backgroundColor = 'yellow';
+    gardenContainer.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (state.tempCount >= 70 && state.tempCount <= 79) {
+    tempCountContainer.style.backgroundColor = 'orange';
+    gardenContainer.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (state.tempCount >= 80) {
+    tempCountContainer.style.backgroundColor = 'red';
+    gardenContainer.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  }
+};
+
+const renderTemp = () => {
+  tempCountContainer.textContent = state.tempCount + ' °F';
+  setTempColor();
+};
+
+renderTemp();
+
+const addTemp = (event) => {
+  state.tempCount += 1;
+  renderTemp();
+};
+
+const subtractTemp = (event) => {
+  state.tempCount -= 1;
+  renderTemp();
+};
+
+const registerEventHandlers = (event) => {
+  // console.log('in registerEventHandelers:', event);
+  const upButton = document.querySelector('#upButton');
+  const downButton = document.querySelector('#downButton');
+  console.log(upButton);
+  upButton.addEventListener('click', addTemp);
+  downButton.addEventListener('click', subtractTemp);
+  console.log(downButton);
+};
+
+registerEventHandlers(undefined);
+
+//AXIOS CODE
+
+//call to the flask app to get weather
+const lat = 47.6038321;
+const lon = -122.330062;
```

Comment:
One of the requirements for Wave 4 was to make two Axios calls and chain them. Rather than hard-coding the lat and lon for Seattle, we wanted you to be able to get this information from the proxy server and then send the information back to the proxy server to get the temperature. This makes the project a yellow.